src/index.js

```diff
@@ -0,0 +1,35 @@
+import './style.css';
+import render from './todosRender';
+import Todos from './todos';
+
+const todosList = new Todos();
+render(todosList);
+
+const addTodoBtn = document.querySelector('.add-btn');
+addTodoBtn.addEventListener('click', () => {
+  const description = document.querySelector('.input-todo').value.trim();
+  const completed = false;
+  const index = todosList.list.length + 1;
+  const id = Math.floor(Math.random() * 1000);
+  const newTodo = {
+    description, completed, index, id,
+  };
+  if (description) {
+    todosList.addTodo(newTodo);
+    render(todosList);
+    document.querySelector('.input-todo').value = '';
+  }
+});
+
+// addTodoBtn.addEventListener("click", () => {
+//   const description = document.querySelector(".input-todo").value.trim();
+//   const completed = false;
+//   const index = todosList.list.length + 1;
+//   const id = Math.floor(Math.random() * 1000);
+//   const newTodo = { description, completed, index, id };
+//   if (description) {
+//     todosList.addTodo(newTodo);
+//     render(todosList);
+//     document.querySelector(".input-todo").value = "";
+//   }
+// });
```

Comment:
- [ ] Kindly remove all commented codes you're not using in your project. It's a good convention to have your code clean and lean. Having codes that are commented on and not meaningful to the project makes your codebase dirty. Please take them off 👍